01week/datatypes.js

```diff
@@ -0,0 +1,55 @@
+console.log(Date())
```

Comment:
I don't know if this is part of the directions, but if not, you don't need this.